ios/ReactNativePayments.m

```diff
@@ -214,6 +218,18 @@ - (void)paymentAuthorizationViewController:(PKPaymentAuthorizationViewController
                                                                                                          }];
 }
 
+// Payment method changes
+ - (void)paymentAuthorizationViewController:(PKPaymentAuthorizationViewController *)controller
+                     didSelectPaymentMethod:(PKPaymentMethod *)paymentMethod
+                                 completion:(nonnull void (^)(NSArray<PKPaymentSummaryItem *> * _Nonnull))completion
+ {
+     self.paymentMethodCompletion = completion;
+     NSString *type = [self paymentMethodTypeToString:paymentMethod.type];
```

Comment:
Do we need to explicitly define `paymentMethodTypeToString` somewhere?

Reply:
yes, paymentMethodTypeToString is defined at line 447 in ReactNativePayments.m